benchmark/text-generation/benchmark.py

```diff
@@ -9,57 +9,46 @@
 from optimum.neuron import NeuronModelForCausalLM
 
 
-def generate(model, input_ids, length):
+def generate(model, input_ids, max_new_tokens):
     start = time.time()
     with torch.inference_mode():
-        output_tokens = model.generate(input_ids, do_sample=False, min_length=length, max_length=length)
+        output_tokens = model.generate(input_ids, do_sample=False, max_new_tokens=max_new_tokens)
     end = time.time()
     return output_tokens, (end - start)
 
 
 def run(model_id, inc_length, max_length, json_path=None):
-    prompts = ["One of my fondest memory"]
+    # Encode the reference prompt
+    with open("./wiki.txt") as f:
+        prompt = f.read()
```

Comment:
You read a 12k lines document and you tokenize everything at once?

Reply:
yes, tokenizer is a brute